src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -255,6 +263,14 @@ private StatusResponsePayload ConvertFrom(DurableOrchestrationStatus status)
             };
         }
 
+        private static DateTime GetDateTimeQueryParameterValue(NameValueCollection queryStringNameValueCollection, string queryParameterName)
+        {
+            var value = queryStringNameValueCollection[queryParameterName];
+            DateTime.TryParse(value, out DateTime dateTime);
+            return dateTime;

```

Comment:
It's not clear what the expected `dateTime` value is if `DateTime.TryParse` fails. I suggest you add a `defaultDateTime` parameter to this method and return that if `DateTime.TryParse` returns `false`.

Reply:
Done